test/e2e/e2e_suite_test.go

```diff
@@ -1,77 +1,372 @@
+	op := fw.GetOperator(namespace)
+	label := op.Deployment.ObjectMeta.Labels
+	pods := &corev1.PodList{}
+	err := fw.ListResources(pods, &client.ListOptions{LabelSelector: labels.SelectorFromSet(label)})
+	require.NoError(fw.GinkgoT, err)
+	return pods.Items[0].Status.ContainerStatuses[0].RestartCount
+}
+
+func (fw *Framework) CleanUpTrenches() {
+	Expect(fw.DeleteAllOfResource(&meridiov1alpha1.Trench{}, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: namespace}})).To(Succeed())
+	Eventually(func() bool {
+		lst := &meridiov1alpha1.TrenchList{}
+		err := fw.ListResources(lst, &client.ListOptions{Namespace: namespace})
+		return err == nil && len(lst.Items) == 0
+	}, 5*time.Second, interval).Should(BeTrue())
+
+}
+
+func (fw *Framework) CleanUpAttractors() {
+	Expect(fw.DeleteAllOfResource(&meridiov1alpha1.Attractor{}, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: namespace}})).To(Succeed())
+	Eventually(func() bool {
+		lst := &meridiov1alpha1.AttractorList{}
+		err := fw.ListResources(lst, &client.ListOptions{Namespace: namespace})
+		return err == nil && len(lst.Items) == 0
+	}, 5*time.Second, interval).Should(BeTrue())
+}
+
+func (fw *Framework) CleanUpVips() {
+	Expect(fw.DeleteAllOfResource(&meridiov1alpha1.Vip{}, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: namespace}})).To(Succeed())
+	Eventually(func() bool {
+		lst := &meridiov1alpha1.VipList{}
+		err := fw.ListResources(lst, &client.ListOptions{Namespace: namespace})
+		return err == nil && len(lst.Items) == 0
+	}, 5*time.Second, interval).Should(BeTrue())
+}
+
+func (fw *Framework) CleanUpGateways() {
+	Expect(fw.DeleteAllOfResource(&meridiov1alpha1.Gateway{}, &client.DeleteAllOfOptions{ListOptions: client.ListOptions{Namespace: namespace}})).To(Succeed())
+	Eventually(func() bool {
+		lst := &meridiov1alpha1.VipList{}
+		err := fw.ListResources(lst, &client.ListOptions{Namespace: namespace})
+		return err == nil && len(lst.Items) == 0
+	}, 5*time.Second, interval).Should(BeTrue())
+}
+
+func (fw *Framework) TrenchReadyCheck(trench *meridiov1alpha1.Trench) {
+	ns := trench.ObjectMeta.Namespace
+	name := trench.ObjectMeta.Name
+	By("checking ipam deployment")
+	Eventually(func(g Gomega) {
+		assertDeploymentReady(g, strings.Join([]string{"ipam", name}, "-"), ns)
+	}, timeout, interval).Should(Succeed())
+
+	By("checking nsp deployment")
+	Eventually(func(g Gomega) {
+		assertDeploymentReady(g, strings.Join([]string{"nsp", name}, "-"), ns)
+	}, timeout, interval).Should(Succeed())
+
+	By("checking proxy deployment")
+	Eventually(func(g Gomega) {
+		assertDaemonsetReady(g, strings.Join([]string{"proxy", name}, "-"), ns)
+	}, timeout, interval).Should(Succeed())
+}
+
+func (fw *Framework) AttractorReadyCheck(trench *meridiov1alpha1.Trench, attractor *meridiov1alpha1.Attractor) {
+	ns := trench.ObjectMeta.Namespace
+	name := trench.ObjectMeta.Name
+
+	By("checking nse vlan deployment")
+	Eventually(func(g Gomega) {
```

Comment:
These "Eventually" with the "By" in the assert functions generates a lot of logs

Reply:
Right. I wanted to see the process of running the tests when that was not working. Now I changed them to comments